src/compiler/compiler.rs

```diff
@@ -162,7 +163,23 @@ where
             Ok(Some(_)) => true,
             _ => false,
         };
-        let result = self.generate_hash_key(&creator, cwd.clone(), env_vars, may_dist, &pool);
+        let rewrite_includes_only = match dist_client {
+            Ok(Some(_)) => dist_client
```

Comment:
Can you just do `Ok(Some(ref client)) => client.rewrite_includes_only(),`?

Reply:
Changed that.